src/App.css

```diff
@@ -1,38 +1,7 @@
 .App {
   text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+  height: 100%;
+  margin: 0;
+  display:flex;
+  flex-direction:column;
+}
```

Comment:
These issues can be avoided by adding insert_final_newline into editorConfig https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#insert_final_newline

Reply:
- [ ] Insert missing empty lines